adapters/repos/db/vector/dynamic/index.go

```diff
@@ -67,6 +67,7 @@ type VectorIndex interface {
 	ContainsNode(id uint64) bool
 	DistancerProvider() distancer.Provider
 	AlreadyIndexed() uint64
+	NewQueryVectorDistancer(queryVector []float32) common.QueryVectorDistancer
```

Comment:
I would change the name of this method to `GetQueryVectorDistancer` or just `QueryVectorDistancer` as it now suggests that this is a constructor

Reply:
It is a constructor  and returns an object of type `common.QueryVectorDistancer`

Reply:
yes, but it's an interface method and that's why I suggested to change the name to `GetQueryVectorDistancer`

Reply:
ok changed